frontend/src/reports/import/Import.svelte

```diff
@@ -39,9 +45,14 @@
       : null;
 
   onMount(() => router.addInteruptHandler(preventNavigation));
-  onMount(() => {
-    files = data;
-  });
+  $: {
+    // Since the user can change e.g. account/name for imported files, we only add any new files here.
+    // This avoids overwriting the users changes
+    const existingFileNames = files.map((file) => file.name);
+    files = files.concat(
```

Comment:
I think we want this to be `files = data.map(...);` to ensure that we have a consistent order of files and also correctly handle removal of files. For each file in `data`, we should check whether it already exists in `files` and if so, take that object instead. [Or, to also correctly handle changing importers, not take the whole existing object from `files` but go over all importers and check whether for that given file.name,importer_name combination we already have an `account` and/or `newName` and copy those over]

Reply:
That's a great suggestion!

I looked at this and took some sort of middle-ground where I just used the `importers` array from the existing entry (if there is one). I think this should cover most cases, I guess in theory the list of importers might have changed, but I guess that won't happen i practice? I think it made the code a bit easier to follow at least.

To get this working I also had to switch to use `router.reload()` when moving and deleting files, I've verified that this works now, and I guess the only reason it wasn't used before was due to this reload-logic not being in place?